docs/UserGuide.md

```diff
@@ -109,16 +109,23 @@ with all attributes containing any of the corresponding keywords in the command.
 Format: `find -PERSON_TYPE [PREFIX/KEYWORDS]`
 
 * All prefixes are optional. Hence, calling `find -PERSON_TYPE` (without any prefixes) will result in all person of the specified type being listed
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* The search is case-insensitive.
+  * e.g `hans` will match `Hans`
+* The order of the keywords does not matter. 
+  * e.g. `Hans Bo` will match `Bo Hans`
+* There are different behaviours regarding the searching of different parameters:
+  * For `NAME`, `EMAIL`, `LOCATION` and `PHONE`, even substrings will be matched 
+    * e.g. `ha` will match `Hans`
+  * For `AGE`, `MEDICAL_HISTORY`, `SPECIALISATION` and `TAGS` only full words will be matched 
```

Comment:
What is the reason for leaving out `MEDICAL_HISTORY` and `SPECIALISATION` for substring search?